### ISSUE TITLE
"Labeled List Summary" Content Discussion

### ISSUE THREAD

**Issue by jaredmorgs** [asciidoctor/asciidoctor.org#447]
http://asciidoctor.org/docs/user-manual/#summary-3 is currently available to create content for.

Use this issue as a tracker for content suggestions, or feel free to provide PRs referencing this ticket to keep track of contributions.

This content is contained in https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/user-manual.adoc

**Comment by jaredmorgs** [comment]
I think it would be useful in some instances to mention now these labeled lists relate back to some commonly-used DocBook constructs like <qandaset>. It may offer a bridge for DocBook users looking at AsciiDoc as a migration strategy.

**Comment by jaredmorgs** [comment]
You could even create a targeted guide for DocBook migrations so folks entrenched in this markup language could migrate easily.

**Comment by mojavelinux** [comment]
I agree that a DocBook to AsciiDoc migration section would be extremely useful. It would be a good place to centralize all these recommendations that we've been spouting off in issue comments. We could also mention [DocBookRx](https://github.com/opendevise/docbookrx), the prescription for your DocBook condition :) (soon to be part of the Asciidoctor organization as soon as we get it release ready).

**Comment by jaredmorgs** [comment]
Great idea. I know that when I was working at Red Hat, the challenge of
migrating out and keeping the Asciidoctor constructs the same as the
DocBook was something that many folks were confused about.

Having DocBookRx is also something that definitely needs to keep maturing.
I'd like to see folks that are using DocBook regularly to somehow have an
easy path to contribute fixes or mappings to this project.

On Sun, 3 Jan 2016 at 10:55 Dan Allen <notifications@github.com> wrote:

> I agree that a DocBook to AsciiDoc migration section would be extremely
> useful. It would be a good place to centralize all these recommendations
> that we've been spouting off in issue comments. We could also mention
> DocBookRx <https://github.com/opendevise/docbookrx>, the prescription for
> your DocBook condition :) (soon to be part of the Asciidoctor organization
> as soon as we get it release ready).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/issues/447#issuecomment-168447563>
> .
>
-- 

Sent from Mobile.
